src/widgets/modals/StocktakeBatchModal.js

```diff
@@ -269,6 +270,17 @@ export const StocktakeBatchModalComponent = ({
     pageTopLeftSectionContainer,
     pageTopRightSectionContainer,
   } = globalStyles;
+
+  useEffect(() => {
+    // Suppress console error only for this component in production
+    // eslint-disable-next-line no-undef
+    console.reportErrorsAsExceptions = __DEV__;
```

Comment:
what are the console errors you're getting?

Reply:
![Screenshot 2023-05-24 at 1 11 43 PM](https://github.com/openmsupply/mobile/assets/13145522/fc1068d3-c3ad-45e9-bdb3-a90e4dfc836c)